.travis.yml

```diff
@@ -11,5 +11,7 @@ notifications:
   email: false
 
 env:
-  secure: "lq31uKdO5F3IfUsfcmPIvmXkp2BOz56pjOAQ4S1RqHF4LoVH6LKMMd3Ocn0C9GyQfomM5NlKTRogFdJvB2iS6piXHej5iSo2/r5tmPxgX3BVMZwV6u9hEuE58U2sA+/NNwWZtivs+1U6XSRNwA2vi2qB8B5lePifCZjxsqL8l1doD4pwH+uDxXoYktberIAKizAzZFi03RvIaXHgPQYBADwrSYY4kbmrl1fL/jfdZx/ouSD055veIVJqQUGwsrdiq5lgtLSR8UfDzWZLQwDTiMLau7fJdE40lwPBAwpwDHPLdf1IGQfMih8y3YC4bQBNMi1w7J/T1nOpABvD8EZhHypYOHO/aYgWMmJM/EEzcytS52r1fieNoV3+GxxdSbOd6DN2D+hrGnZ34c3sQ8UZWXv6/PZtBBIcOsw132cQ35dOSXx78khg129IauW+RiQ5bRglZMr0Su6Pz3AL7az1CG0IDzzlJC+vmcobPLDTHJWpXIO2dyv/NyzfD0d5tYc9I1Y2/KeLUBxEm6x2GZw/dSQRih9YsZc7FXTyReNE5TQ8qGrDhpKGswr4c8MxW/3Dyktb0VwO2TlCfXwB0QKyTdd5QFSNYZ6OJyn1uPY/jHpTEzYxGAIVVFbWBwyv6NkjsIuFzhyLaOtmp6klHTdqcdFSExtIpL+8QoRmvWM06dc="
+  global:
+    - WOWZA_HONEYBADGER_ENV=travis_honeybadger_env
```

Comment:
now setting these env vars in project settings on travis-ci.org, removed them from `.travis.yml`